csharp/autobuilder/Semmle.Autobuild.CSharp/DotNetRule.cs

```diff
@@ -152,7 +156,18 @@ private static BuildScript DownloadDotNet(IAutobuilder<AutobuildOptionsShared> b
                 validGlobalJson = true;
             }
 
-            return validGlobalJson ? installScript : BuildScript.Failure;
+            if (validGlobalJson)
+            {
+                return installScript;
+            }
+
+            if (ensureDotNetAvailable)
+            {
+                const string latestDotNetSdkVersion = "8.0.101";
```

Comment:
Should we consider to add a "global" constant with this version number somewhere and maybe add a comment to it (that we need to update this when we update .NET version). Maybe also add it to the .NET update guide in the C# team repo.